packages/core/diagnostic/src/diagnostic.js

```diff
@@ -1,5 +1,4 @@
 // @flow strict-local
-import type {FilePath} from '@parcel/types';
```

Comment:
Avoiding circular dependency... Perhaps we should consider whether the diagnostic types should move into the core types package and only the implementation should stay in this package?